src/datamap.js

```diff
@@ -2,7 +2,7 @@ import iota from "./services/iota";
 import Encryption from "./utils/encryption";
 
 import { FILE } from "./config";
-import forge from "node-forge";
+import util from "node-forge/lib/util";
```

Comment:
Is is possible to just import hexToBytes?

Reply:
unfortunately, no. They are exporting the entire util object.